src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -394,6 +394,7 @@ export default function MarkdownEditor({
       ) : (
         <TextArea
           aria-label={label}
+          placeholder={label}
```

Comment:
A placeholder and a label are different in general. A label is a field name, whereas the placeholder is an example value or a hint. In this case if we had visible labels we'd likely use just "Comment" or "Reply" for the label and "Enter comment..." / "Enter reply..." (maybe without "...") as the placeholder.

Having said this, in this particular instance the current version looks fine and reads in a sensible way with VoiceOver. When the label and the placeholder are the same, I notice that VoiceOver only reads the value once.

Reply:
> When the label and the placeholder are the same, I notice that VoiceOver only reads the value once.
orca does the same.